Modules/_sqlite/connection.c

```diff
@@ -619,6 +619,28 @@ _pysqlite_build_py_params(sqlite3_context *context, int argc,
     return NULL;
 }
 
+static void
+_pysqlite_result_error(sqlite3_context *context, const char *msg)
```

Comment:
Nitpick: The scope of this function is file only; I don't like polluting the namespace unnecessarily, so I'd prefer to drop the `_pysqlite_` prefix.

Reply:
Most of static functions in this file (and in other files in this directory) start with prefixes `_pysqlite_` or `pysqlite_`.

Although we can omit it in new code.